src/Fetcher/ConfTechFetcher.php

```diff
@@ -146,6 +150,14 @@ public function denormalizeConferences(array $rawConferences, string $source, Ta
             $conference->setSiteUrl($rawConference['url']);
             $conference->addTag($tag);
 
+            $excluded = false;
+            foreach ($this->excludedTags as $excludedTag) {
+                if (fnmatch($excludedTag->getName(), $rawConference['name'], FNM_CASEFOLD)) {
+                    $excluded = true;
```

Comment:
You can break the loop after this because it will never happen to reset `$excluded` to false.